controller/helper/process-trial.js

```diff
@@ -27,14 +28,40 @@ function processTrial (trial) {
         IRBID: trial.IRBID,
         start: startDate.format('L'),
         end: endDate.format('L'),
-        recruitedPercent: null,
-        unrecruitedPercent: null,
-        activePercent: null,
-        completedPercent: null,
+        recruitedPercent: allPercent.recruitedPercent,
+        unrecruitedPercent: allPercent.unrecruitedPercent,
+        activePercent: allPercent.activePercent,
+        completedPercent: allPercent.completedPercent,
         compliantPercent: null,
         noncompliantPercent: null,
         noncompliantCount: null
     };
 }
 
+/**
+ * Takes in a Trial model and processes to provide percent calculation
+ * @param {Trial} trial - a single Trial object
+ * @returns {Object} calculated Percent
+ */
+function processPercent (trial) {
+    const percent = 100;
+    const zeroPercent = 0;
+
+    if (trial.recruitedCount > zeroPercent) {
```

Comment:
I guess we need to handle the condition where no patients are yet recruited in the trial. which means that trial.recruitedCount = 0 then how does the code flow handle the bars? Because we had similar issues with donut chart and it resulted into error 404